stdlib/sql/sql.flux

```diff
@@ -1,7 +1,421 @@
 package sql
```

Comment:
Can you add a package doc comment? See https://docs.influxdata.com/influxdb/cloud/reference/flux/stdlib/sql/